application.go

```diff
@@ -225,3 +229,13 @@ func (impl *applicationImpl) ListApplications(ctx context.Context, opts *Resourc
 
 	return &applicationListIterator{iter: iter}, nil
 }
+
+func (impl *applicationImpl) DebugApplicationResources(ctx context.Context, appID ID, opts *ResourceGetOptions) (string, error) {
+	var rawData json.RawMessage
+	uri := path.Join(_apiPathPrefix, "controlplanes", opts.ControlPlane.ID.String(), "application", appID.String())
```

Comment:
The debug api is served in `/api/v1/debug/config/controlplanes/{control_plane_id}/{type}/{obj_id}`.